WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/SpeakerApp.kt

```diff
@@ -46,14 +54,18 @@ import kotlinx.coroutines.launch
  */
 @OptIn(ExperimentalHorologistApi::class)
 @Composable
-fun SpeakerApp() {
+fun SpeakerApp(player: Player) {
     MaterialTheme {
```

Comment:
Do we have AppScaffold somewhere?

Reply:
yeah,  [Speaker app](https://github.com/android/wear-os-samples/pull/1094/files#diff-667f2835722c1429eaeb80be39a23315fd175d0c299403e0bd99ec7c73825d1dR86)